client/src/components/Submissions/Submissions.jsx

```diff
@@ -0,0 +1,151 @@
+  const submitDisabled = Object.values(uploadedFiles).every((f) => {
+    return status[f.id] === 'SUCCESS' || status[f.id] === 'ERROR';
+  });
+
+  const selectFiles = (acceptedFiles) => {
+    const newFiles = [];
+    const newAcceptedFiles = acceptedFiles.filter(
+      (af) =>
+        uploadedFiles.filter(
+          (uf) => uf.data.path === af.path && uf.data.size === af.size
+        ).length === 0
+    );
+    newAcceptedFiles.forEach((file) => {
+      newFiles.push({ data: file, id: uuidv4() });
+    });
+    setUploadedFiles((files) => [...files, ...newFiles]);
+  };
+
+  const removeFiles = () => {
+    setUploadedFiles((uploadedFiles) => []);
+  };
+
+  const onRejectedFiles = (oversizedFiles) => {
+    const newFiles = [];
+    const newRejectedFiles = oversizedFiles.filter(
+      (of) =>
+        rejectedFiles.filter((rf) => rf.data.path === of.path).length === 0
+    );
+    newRejectedFiles.forEach((file) => {
+      newFiles.push({ data: file, id: uuidv4() });
+    });
+    setUploadedFiles((files) => [...files, ...newFiles]);
+    setRejectedFiles((files) => [...files, ...newFiles]);
+  };
+
+  return (
+    <div className={styles['container']}>
+      <div disabled={dropZoneDisabled}>
+        <FileInputDropZone
+          onSetFiles={selectFiles}
+          onRejectedFiles={onRejectedFiles}
+          maxSize={2147483648}
+          maxSizeMessage="Max File Size: 2GB"
+        />
+      </div>
+      {uploadedFiles.length > 0 && (
+        <div>
+          <div className={styles['listing']}>
+            <span className={styles['listing-header']}>
+              File Ready for Upload:
+            </span>
+            <DataFilesUploadModalListingTable
+              uploadedFiles={uploadedFiles}
+              rejectedFiles={rejectedFiles}
+              setUploadedFiles={setUploadedFiles}
+              submission={true}
+            />
+          </div>
+          {submitDisabled ? (
+            <Button
+              className={styles['listing-button']}
+              type="secondary"
+              size="long"
+              onClick={removeFiles}
+            >
+              Clear
+            </Button>
+          ) : (
+            <Button
+              className={styles['listing-button']}
+              type="primary"
+              size="long"
+              onClick={uploadStart}
+            >
+              Upload File(s)
+            </Button>
+          )}
+        </div>
+      )}
+    </div>
+  );
+};
+
+const Submissions = () => {
+  return (
+    <Section
+      bodyClassName="has-loaded-dashboard"
+      messageComponentName="SUBMISSIONS"
+      introMessageText={
+        <span>
+          For security reasons, files uploaded here cannot be accessed after
+          uploading.{' '}
+          <strong>
+            If a file exceeds 2GB, refer to the{' '}
+            <Link to="/help/data-transfer-guide/" className="wb-link">
+              Large Data Transfer Guide
+            </Link>
```

Comment:
Ah you know, I think this needs to be an anchor tag since it's not pointing to another portion of the client.
```suggestion
            <a href="/help/data-transfer-guide/" className="wb-link">
              Large Data Transfer Guide
            </a>
```